fe/fe-core/src/main/java/com/starrocks/sql/ast/StatementBase.java

```diff
@@ -48,7 +48,8 @@ public enum ExplainLevel {
         VERBOSE,
         // True if the describe_stmt print costs information, if `isCosts` is true, `isExplain` must be set to true.
         COST,
-        OPTIMIZER
+        OPTIMIZER,
+        VERBOSE_OPTIMIZER
```

Comment:
use optimizer directly~ don't need add a verbose

Reply:
> use optimizer directly~ don't need add a verbose

Yes enhance the trace optimizer is better.

Reply:
> the performance of VERBOSE_OPTIMIZER mode is worse

`trace optimizer` is more like a debug tool. I think the info provided by it is more important than its performance.